docs/help-topics/rhtap-import-configure-component/rhtap-import-configure-component.yml

```diff
@@ -11,18 +11,23 @@
     
     Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too.
 
-    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL.
-    - **Runtime**: Did we detect the right one? If so, you should notice that we suggest appropriate deployment options. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
+    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL. Save any changes to the name by clicking the checkmark. Discard the changes and exit editing mode by clicking the X.
```

Comment:
There is no checkmark anymore, it's a simple text input field.
![image](https://github.com/RedHatInsights/quickstarts/assets/14068621/f0b29a3f-bb57-4bbf-aa60-bb1a4502efaf)

Reply:
^ This is in reference to renaming a component. If they include a character that we don't support, an error message is displayed immediately. If they name it the same as another component, when they try to advance past this screen, they'll see an error and have to update the component name. There's no "undoing" a name change anymore - they retype it in the field.

Reply:
This information about saving/cancelling still needs to be updated to describe the current UI behavior.